en/releases/1.15.md

```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
+  New parameter: VT_QC_ALT_LOSS
+- [FMUv6X]: Increase TELEM2 RX buffer size to increase throughput. [PX4-Autopilot#21660](https://github.com/PX4/PX4-Autopilot/pull/21660)
+- Feature Add power module selector for Auterion power modules. [PX4-Autopilot#22073](https://github.com/PX4/PX4-Autopilot/pull/22073)
+- lights: Add LP5562 RGBLED driver. [PX4-Autopilot#21649](https://github.com/PX4/PX4-Autopilot/pull/21649)
+- Enable EKF2 on KakuteH7 / KakuteH7v2 / KakuteH7mini. [PX4-Autopilot#21961](https://github.com/PX4/PX4-Autopilot/pull/21961)
+- cubepilot: Add support for 4. variant of Orange+. [PX4-Autopilot#22039](https://github.com/PX4/PX4-Autopilot/pull/22039)
 
 ### Common
 
 - `SYS_MC_EST_GROUP` has been removed and there are now dedicated parameters [EKF2_EN](../advanced_config/parameter_reference.md#EKF2_EN), [ATT_EN](../advanced_config/parameter_reference.md#ATT_EN), and [LPE_EN](../advanced_config/parameter_reference.md#LPE_EN) for [Switching State Estimators](../advanced/switching_state_estimators.md).
   Most setups should enable [EKF2_EN](../advanced_config/parameter_reference.md#EKF2_EN) (the default).
   Changed in [PX4-Autopilot#22567](https://github.com/PX4/PX4-Autopilot/pull/22567).
+- [Experimental] Zenoh pico Support [PX4-Autopilot#22017](https://github.com/PX4/PX4-Autopilot/pull/22017)
+- [Bootloader] add force-erase option and bootloader version [PX4-Autopilot#22777](https://github.com/PX4/PX4-Autopilot/pull/22777)
+- [uORB] Allow for more than 255 uORB messages [PX4-Autopilot#21923](https://github.com/PX4/PX4-Autopilot/pull/21923)
+- Feature: Improve mission resume (e.g. replay gimbal and camera commands of a survey) [PX4-Autopilot#21710](https://github.com/PX4/PX4-Autopilot/pull/21710)
+- Feature: Resume mission with flight speed from previous mission items [PX4-Autopilot#21714](https://github.com/PX4/PX4-Autopilot/pull/21714)
+- Feature: Warn user when navigation failure is imminent [PX4-Autopilot#21876](https://github.com/PX4/PX4-Autopilot/pull/21876)
+- Navigator: Geofence improvements [PX4-Autopilot#22394](https://github.com/PX4/PX4-Autopilot/pull/22394)
+- Improvement: Commander: trigger warning when arming denied due to check failure. [PX4-Autopilot#22885](https://github.com/PX4/PX4-Autopilot/pull/22885)
+- Improvement: Commander: make failsafe warnings shorter [PX4-Autopilot#22886](https://github.com/PX4/PX4-Autopilot/pull/22886)
+- Support for a remote parameter database. [PX4-Autopilot#22836](https://github.com/PX4/PX4-Autopilot/pull/22836)
+- New platform independent UART interface. [PX4-Autopilot#21723](https://github.com/PX4/PX4-Autopilot/pull/21723)
 
 ### Control
 
-- [offboard][ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
+- [offboard] [ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
   Added in [PX4-Autopilot#22222](https://github.com/PX4/PX4-Autopilot/pull/22222).
 
 ### Estimation
```

Comment:
@bresch please see below, I went trough the changelog without seeing your discord message which I will be incorporating into this. 

For now can you do a quick review here please? inline comments are ok